kiwi/schema/kiwi.rnc

```diff
@@ -1644,6 +1644,15 @@ div {
             sch:param [ name = "attr" value = "installpxe" ]
             sch:param [ name = "types" value = "oem" ]
         ]
+    k.type.checkiso.attribute =
+        ## Specifies if the bootloader menu should provide an"
+        ## mediacheck entry to verify ISO integrity or not.
+        ## Disabled by default and only available for x86 arch family."
+        attribute checkiso { xsd:boolean }
```

Comment:
Yes that's true, it sounds more like a buildtime check. I will update the attribute name.